packages/components/tests/dummy/app/router.js

```diff
@@ -30,6 +30,7 @@ Router.map(function () {
     this.route('button-set');
     this.route('card');
     this.route('code-block');
+    this.route('dialog-primitive');
```

Comment:
TBD if we want to have it under the "Utilities" categories as for the other "primitives" (personally, I don't dislike at all the `DialogPrimitive` name).

Reply:
Maybe make a new "Primitives" category?

Reply:
Yeah, maybe.